doc/cephfs/cephfs-journal-tool.rst

```diff
@@ -237,3 +241,12 @@ Example: event mode
     # cephfs-journal-tool event get binary --path bin_events
     Wrote output to binary files in directory 'bin_events'
 
+.. _cephfs_journal_tool_options:
+
+Options
+~~~~~~~
+
+* ``--rank=<filesystem>:{mds-rank|all}`` Used to specify the filesystem, the MDS rank or all ranks.
+* ``--journal=<mdlog|purge_queue>`` The journal type. The default value is ``mdlog``, which is the
+  only option that supports event mode. When ``purge_queue`` is used, the journal scanner creates
+  a PurgeItem object and add it to EventMap for processing.
```

Comment:
s/and add it/and adds it/

This is much clearer than it was before.

Reply:
Updated